app/javascript/mastodon/features/terms_of_service/index.jsx

```diff
@@ -30,7 +30,7 @@ class TermsOfService extends React.PureComponent {
         <div className='scrollable privacy-policy'>
           <div className='column-title'>
             <h3>Terms of Service</h3>
-            <p>Version 0.1, effective March 31, 2023</p>
+            <p>Version 0.2, effective October 4, 2023</p>
```

Comment:
Amended this just to bring it to the reviewers' attention - should we update the date and version here or leave it unchanged?